src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -33,6 +34,11 @@ public class IntakeSubsystem extends SubsystemBase {
 
 	private final CANSparkFlex feederMotor;
 
+	// Sensors
+	private final DigitalInput intakeSensor;
```

Comment:
The robot currently doesn't have an intake sensor, so should remove this & the getter method below

Reply:
Rest of the code looks fine.  Nice work!

Reply:
Remove only the intake sensor and it's getter method, or the rest of them as well?

Reply:
Just the intake sensor. The other sensors are good & should stay